replay_parser/src/packet_parser/events/entity_method/avatar_methods/update_arena/mod.rs

```diff
@@ -40,6 +44,8 @@ pub enum ArenaUpdateData {
     Period(Period),
     VehicleDescr(VehicleDescr),
     FogOfWar(FogOfWar),
+    VehicleAdded(VehicleData),
```

Comment:
Do you have a replay where this event occurs? My replays do not seem to have this event

Reply:
Never mind! I found it.